plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -213,7 +213,16 @@ jQuery( function( $ ) {
 		// But only zoom if the img is larger than its container.
 		if ( zoomEnabled ) {
 			var zoom_options = $.extend( {
-				touch: false
+				touch: false,
+				callback: function() {
+					var zoomImg = this;
+					
+					setTimeout( function() {
```

Comment:
`setTimeout` is required to wait for the `jquery.zoom.js` plugin to complete adding the attributes to the image element so we can manipulate them.